com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -258,6 +265,20 @@ struct BRDFData
     half roughness2MinusOne;    // roughness^2 - 1.0
 };
 
+BRDFData InitializeBRDFData()
+{
+    BRDFData r;
```

Comment:
suggestion: you can do BRDFData r = (BRDFData)0; for same result and less verbose.

Reply:
Yes you're right. I didn't know that works in hlsl.
It appears to do element-wise assignment, so Struct = (Struct)-32 would set every element in the struct to -32.
I'll remove the function.